server/database/schemas/ListedSong.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose")
+const listedSongSchema =new mongoose.Schema({
+  
+creator:{
```

Comment:
Please remove the unnecessary blank line on line 3 and line 20. Also please fix your indentation throughout lines 4 through 19.